cms/static/js/views/video_thumbnail.js

```diff
@@ -216,6 +278,145 @@ define(
                 this.$('.thumbnail-wrapper').attr('class', 'thumbnail-wrapper {action}'.replace('{action}', action));
             },
 
+            validateImageFile: function(imageFile) {
+                var errorMessage = '';
+
+                if (!_.contains(this.getVideoImageSupportedFileFormats().machine, imageFile.type)) {
+                    errorMessage = StringUtils.interpolate(
+                        // Translators: supportedFileFormats will be like .bmp, gif, .jpg or .png.
+                        gettext(
+                            'This image file type is not supported. Supported file types are {supportedFileFormats}.'
+                        ),
+                        {
+                            supportedFileFormats: this.getVideoImageSupportedFileFormats().humanize
+                        }
+                    );
+
+                } else if (imageFile.size > this.getVideoImageMaxSize().machine) {
+                    errorMessage = StringUtils.interpolate(
+                        // Translators: maxFileSizeInMB will be like 2 MB.
+                        gettext('The selected image must be smaller than {maxFileSizeInMB}.'),
+                        {
+                            maxFileSizeInMB: this.getVideoImageMaxSize().humanize
+                        }
+                    );
+                } else if (imageFile.size < this.getVideoImageMinSize().machine) {
+                    errorMessage = StringUtils.interpolate(
+                        // Translators: minFileSizeInKB will be like 2 KB.
+                        gettext('The selected image must be smaller than {minFileSizeInKB}.'),
```

Comment:
@mushtaqak `The selected image must be smaller than` -> `The selected image must be larger than`

Credit goes to @Qubad786 for catching this :)